classroom.rb

```diff
@@ -0,0 +1,11 @@
+class Classroom
+  def initialize(label, _students)
+    @label = label
+    @_students = []
+  end
+  attr_accessor :label
+
+  def add_students(student)
+    @students.push(student)
+  end
```

Comment:
Great job @Therealadetunji 🤝

- [ ] Kindly make sure this method serves as the `has-many` side (a classroom has many students). You can check this [link](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/articles/oop_relationships_by_examples.md#both-ways-relationships) for more information.
- [ ] You may also add the attribute reader for students `attr_reader :students`.